pyso/stack_search_module.py

```diff
@@ -8,7 +8,7 @@
 import async_timeout
 from .helpers import *
 
-class pyso:
+class PySo:
     def __init__(self, access_token, key='uKDJS)Ck32rrSYrweVOoOQ(('):
```

Comment:
Yes, a key is pretty much public & will stay the same for a long while. Unless this has recently changed, the documentation itself states that it doesn't matter if someone else gets hold of it / it's publically available. Having it "hard-coded" / public allows for easy use using the default application. :)